packages/cli/src/utils.ts

```diff
@@ -252,18 +251,11 @@ const getAndroidDetails = async (
   const minSdk = new RegExp(
     AaptPrefixes.sdkPrefix + AaptPrefixes.quoteRegex
   ).exec(stdout);
-  const permissions = new RegExp(
-    AaptPrefixes.permissionPrefix + AaptPrefixes.quoteNonLazyRegex
-  ).exec(stdout);
+  const permissions = [...stdout.matchAll(/uses-permission: name='(.*)'/g)];
   const locales = new RegExp(
     AaptPrefixes.localePrefix + AaptPrefixes.quoteNonLazyRegex
   ).exec(stdout);
 
-  let permissionArray = Array.from(permissions?.values() ?? []);
-  if (permissionArray.length >= 2) {
```

Comment:
It'd be good to get some context from @creativedrewy on what the purpose of slicing the first element from the permissions list was.

Reply:
The existing code only matches 1 instead of all possible substrings.
`/g` was needed in the regex and change from `exec` to `matchAll`

A run of `aapt2 dump badging` gives outtput like this for permission

```
uses-permission: name='android.permission.INTERNET'
uses-permission: name='android.permission.WRITE_EXTERNAL_STORAGE'
```

Current regex matching (`uses-permission: name='(.*)'`)  was giving an array where the first element was the entire line (`uses-permission: name='android.permission.INTERNET'
`) and second element was the subgroup (`(.*)`) in regex (i.e. `android.permission.INTERNET`)

Slicing index 1 got the correct permission value however only for 1 permission not all permissions.
The example apk added only has 1 permission and possibly the reason why existing code seems like working correctly and not properly tested.

New code replaces the slice with flatMap to pick the subgroup from regex 
```
permissions.flatMap(permission => permission[1]),
```